angular/projects/common/src/app/entity/entity-detail/entity-detail.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="grid-section section-1">
   <div class="grid-item">
-    <label class="grid-item-name">Entity Type</label>
+    <span class="grid-item-name">Entity Type</span>
```

Comment:
Shouldn't these be made into spans?  Is it the grid layout that is causing the problem?